www.foia.gov/_sass/modules/_form_group.scss

```diff
@@ -18,3 +20,17 @@
   font-size: $small-font-size;
   color: $color-gray;
 }
+
+// field-description is auto-generated by react-jsonschema-forms
+.usa-form-hint,
+.field-description {
```

Comment:
I think we should put the react-jsonschema-forms overrides in their own file, but we can refer to what's here with `@extend`. For example:
```scss
// _react-jsonschmema-forms.scss
.field-description {
  @extend .usa-form-hint;
}
```

.. Or vice versa.

Reply:
hmm, maybe... idk.

Reply:
That's probably more robust and less error prone than relying on comments.

Reply:
Lulz, I actually had this set up this way already, and just forgot to include it in the modules.